### ISSUE TITLE
tank_traps  invisible

### ISSUE THREAD

**Issue by BertMule** [gravitystorm/openstreetmap-carto#4551]
Various tank_traps seem to have suddenly become invisible.
Around [this area](https://www.openstreetmap.org/#map=18/52.17420/4.38801).

As far as I know they were visible until recently, I don't see recent related changes in the history.
Has rendering changed?

**Comment by HolgerJeromin** [comment]
There was never a rendering of them.
ref #4498

**Comment by BertMule** [comment]
Yes, I just concluded that might have been the case.

I will have to revert them to a 'wall', as they partially were. 
Including a part that wasn't, to make them all visible.
That's 'mapping for rendering'. Undesirable.

So, requesting to render them.

**Comment by imagico** [comment]
Closing as duplicate of #4498.

For info: `barrier=tank_trap` + `historic=yes` has about 150 uses but this tagging is not consensus among mappers for features with no present day function as tank traps because it competes with lifecycle prefixes (like `abandoned:military=tank_trap` - about the same number of uses).

**Comment by BertMule** [comment]
So what are you saying?
Should the one be replaced by the other?

**Comment by imagico** [comment]
No, mappers are free to choose the tagging they like, as a project we try to not have no position on which tagging is *right*.  What i am saying is that there is no consensus among mappers what tagging to use for historic tank traps with no present day use as such.  With several competing tagging schemes in use we are usually very careful when it comes to introducing new features to rendering.  But in any case any discussion on that should go to #4498.

**Comment by Adamant36** [comment]
> Instead of mappers falling back to 'walls' or the like, which will, and already has, happened.

Just as general point, I don't think tagging tank traps that are walls as such is necessarily wrong since they are still walls. Like if someone builds a wall around their house to keep out animals it's still tagged as a wall the same as one built for any other purpose is. 99% of people walking by a wall that was built to stop tanks probably isn't going to know that's what it was originally for anyway. In the meantime there seems to be a "wall=*" tag that might be usable to tag what kind of wall it is, but the tagging is all over the place. So I don't think it's helpful, but you might look into something what tags there are along those lines to accomplish what you want.
